style.css

```diff
@@ -129,6 +129,12 @@ body {
   border-radius: 10px;
   font-size: 13px;
   margin-right: 10px;
+  transition: all 4s;
```

Comment:
- [ ] Kindly change the contrast to be at least `4.5` for these failed elements because they failed in the contrast check
![screencapture-aditus-io-button-contrast-checker-rawcdn-githack-com-2022-03-29-at-16-24-26-535-2022-03-29-18_38_02](https://user-images.githubusercontent.com/62290545/160663118-3e62f06d-2025-408b-b33f-d0af64dcef3a.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)